docs/FeatureManagement/Clients/Microsoft.FeatureManagement.AspNetCore.md

```diff
@@ -0,0 +1,367 @@
+# ASP.Net Core Feature Flags
+
```

Comment:
As discussed, you can add a link to API reference for Microsoft.FeatureManagement: https://go.microsoft.com/fwlink/?linkid=2091700.

Reply:
Done.